.tool-versions

```diff
@@ -1 +1 @@
-golang 1.20
+golang 1.21.3
```

Comment:
Specifically, here we want to ensure the same toolchain is used

Reply:
According to https://go.dev/doc/toolchain

> If the toolchain line is omitted, the module or workspace is considered to have an implicit toolchain goV line, where V is the Go version from the go line.

So I would set this version to be the same as the go version in go mod if we don't use the toolchain

Reply:
See: https://github.com/mongodb/mongodb-atlas-cli/pull/2356#discussion_r1356620224